### DISCUSSION TITLE
Mixed audio condition?

### DISCUSSION THREAD

**Discussion by phillypro** [WarmUpTill/SceneSwitcher#606] [Ideas]
I kid you not 

i am attempting to use your plugin to replace a media console that does automatic video follows audio switching

the roland vr 50hd mk ii 

it currently works like this....

you can assign a mic to a cam input...and if that mic goes above a threshold it switches

but theres also an option for if more than one mic goes above that threshold (mix)  then do this action

ususally i have a wide shot set for that

is there any way to recreate that setting of "mix"  for this switcher?

**Comment by WarmUpTill** [comment]
This guide in the wiki contains an example, which might help you to set this up:
https://github.com/WarmUpTill/SceneSwitcher/wiki/Variables#example-podcast

Let me know if you have any questions! :)

**Reply by phillypro** [reply]
WHERE WERE YOU MONTHS AGO!!!!! 

you are a godsend my friend....i cant wait to try this.....once i sell this massive piece of equipment and get a PC
